play-services-base/core/src/main/res/layout/preference_switch_bar.xml

```diff
@@ -36,7 +36,7 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center_vertical"
         android:background="@null"
-        app:thumbTint="?android:attr/textColorPrimaryInverse"
-        app:trackTint="?android:attr/textColorSecondaryInverse"
+        app:thumbTint="@android:color/white"
+        app:trackTint="@android:color/darker_gray"
```

Comment:
These won't work when adjusting to dark/light theme, no?

Reply:
Looks fine in both themes

![image](https://github.com/microg/GmsCore/assets/46414010/8b732efb-2d10-4c78-b95c-799f930f3486)
![image](https://github.com/microg/GmsCore/assets/46414010/db32bc02-2d9c-4ea9-851c-6a048dd50e53)

Reply:
![Screenshot_20240210_114110](https://github.com/microg/GmsCore/assets/46414010/1fe8a79e-a63c-4473-91f5-9f214adfd7a8)
![Screenshot_20240210_114110](https://github.com/microg/GmsCore/assets/46414010/32f1c95f-31be-4891-b521-bbf3ce98f149)
![Screenshot_20240210_114135](https://github.com/microg/GmsCore/assets/46414010/2a96deba-ea72-4dcf-94cd-3ea17250a265)
![Screenshot_20240210_114123](https://github.com/microg/GmsCore/assets/46414010/35de0e5f-b566-44ea-aa52-63358c21c9b6)

Reply:
@mar-v-in apologies for the late reply. Here are the screenshots of the bar in both light and dark theme.